.github/workflows/create-release.yaml

```
@@ -85,29 +87,26 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: autotelic/action-wait-for-status-check@6556cf50c8fb6608412945382eae73581f56cbb4
-        name: Build image
-        id: wait-for-build
+      - name: Image build
+        uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          statusName: "rel-istio-build"
-          intervalSeconds: "30"
-          timeoutSeconds: "1800"
-      - name: Check that module build status is success
-        if: steps.wait-for-build.outputs.state != 'success'
-        run: exit 1
-
-      - uses: autotelic/action-wait-for-status-check@6556cf50c8fb6608412945382eae73581f56cbb4
-        name: Verify build of experimental image
-        id: wait-for-experimental-build
+          name: istio-manager
+          dockerfile: Dockerfile
+          context: .
+          build-args: |
+            VERSION=${{ github.event.inputs.name }}
+          tags: "${{ github.event.inputs.name }}"
+
+      - name: Image build - experimental
```

Should we use the same name we use on main?
```suggestion
      - name: Build manager image - experimental
```

---

looks better, let's do that